### DISCUSSION TITLE
invalid response header

### DISCUSSION THREAD

**Discussion by zh-github-1** [seaweedfs/seaweedfs#3994] [Q&A]
The response header has an attribute of Content-Disposition: inline; filename="%{#context['com.opensymphony.xwork2.dispatcher.HttpServletResponse'].addHeader('X-RES',41706*40530)}

**Comment by chrislusf** [comment]
need to know how to reproduce

**Reply by zh-github-1** [reply]
My seaweedfs version is version 1.27 linux amd64.
My Capture data:
Frame 74: 441 bytes on wire (3528 bits), 441 bytes captured (3528 bits) on interface \Device\NPF_{XXXX}, id 0
Transmission Control Protocol, Src Port: 9301, Dst Port: 56138, Seq: 1, Ack: 245, Len: 387
Hypertext Transfer Protocol
    HTTP/1.1 200 OK\r\n
        [Expert Info (Chat/Sequence): HTTP/1.1 200 OK\r\n]
            [HTTP/1.1 200 OK\r\n]
            [Severity level: Chat]
            [Group: Sequence]
        Response Version: HTTP/1.1
        Status Code: 200
        [Status Code Description: OK]
        Response Phrase: OK
    Accept-Ranges: bytes\r\n
    Content-Disposition: inline; filename="%{#context['com.opensymphony.xwork2.dispatcher.HttpServletResponse'].addHeader('X-RES',41706*40530)}\000b"\r\n
        [Expert Info (Warning/Undecoded): Trailing stray characters]
            [Trailing stray characters]
            [Severity level: Warning]
            [Group: Undecoded]
    Content-Encoding: gzip\r\n
    Content-Length: 23\r\n
    Etag: "aef6e3b8"\r\n
    Last-Modified: Sat, 19 Nov 2022 14:21:23 GMT\r\n
    Date: Mon, 21 Nov 2022 04:24:26 GMT\r\n
    Content-Type: application/x-gzip\r\n
    \r\n
    [HTTP response 1/1]
    [Time since request: 0.007554000 seconds]
    [Request in frame: 72]
    [Request URI: http://myremoteip:9333/6,40d3155e3307]
    Content-encoded entity body (gzip): 23 bytes -> 0 bytes
    File Data: 0 bytes

**Reply by chrislusf** [reply]
this depends on how the file is uploaded

**Reply by zh-github-1** [reply]
> this depends on how the file is uploaded

upload by java-client

**Reply by chrislusf** [reply]
need working code to reproduce it
